cmd/zed/manage/command.go

```diff
@@ -15,14 +15,14 @@ import (
 var Cmd = &charm.Spec{
 	Name:  "manage",
 	Usage: "manage",
-	Short: "continuously run compaction and other maintenance tasks on a lake",
+	Short: "run compaction and other maintenance tasks on a lake",
 	New:   New,
 }
 
 type Command struct {
 	*root.Command
 	logFlags logflags.Flags
-	once     bool
+	monitor  bool
 	config   lakemanage.Config
```

Comment:
Nit: Swap these to match the order in which New initializes them.
```suggestion
	config   lakemanage.Config
	monitor  bool
```